### PR TITLE
fix(main/telegram-bot-api): Switch to tag version control

### PR DESCRIPTION
**Pull request by Sarisan**

### CODE REVIEW

**Comment by Sarisan** [comment]
Wasn't supposed to be opened now.

File: packages/telegram-bot-api/build.sh [hunk 1]
```diff
@@ -1,44 +1,17 @@
...
-TERMUX_PKG_SRCURL=git+https://github.com/tdlib/telegram-bot-api
-_COMMIT=12bbe26692b8c6e347aaf3e32e0e9d11b86bb56d
-_COMMIT_DATE=2024.07.30
-TERMUX_PKG_VERSION=${_COMMIT_DATE//./}
-TERMUX_PKG_SHA256=6c0dddef10c0e1050a0f9840895d71830a867bb1932cd150d1d56b1e9510fc18
-TERMUX_PKG_AUTO_UPDATE=false
-TERMUX_PKG_GIT_BRANCH=master
-TERMUX_PKG_HOSTBUILD=true
+TERMUX_PKG_MAINTAINER="Maria Lisina @Sarisan"
+TERMUX_PKG_VERSION="7.8"
```
Comment (licy183) [inline packages/telegram-bot-api/build.sh:5]: Epoch should be bumped to 1 as `7.8` < `20240730`

```suggestion
TERMUX_PKG_VERSION="1:7.8"
```

**Review by licy183 — CHANGES_REQUESTED** [review]

**Comment by Biswa96** [comment]
The change is using a fork of the actual repository. I am not sure if that should be done.

**Comment by Sarisan** [comment]
> The change is using a fork of the actual repository. I am not sure if that should be done.

I don't see any other way to implement auto updates for this packages, other than if there were tags, it would still require manipulations with git and submodule. I can actually try to ask developer to tag releases, but I doubt they will.

**Comment by Biswa96** [comment]
Not all packages can be auto-updated. I would recommend maintaining the current version type if upstream does not have proper tags.

**Comment by Sarisan** [comment]
I will wait for developer response, closing this till then.
https://github.com/tdlib/telegram-bot-api/issues/623

**Pull request closed without merging**